deploy/charts/cert-manager/values.yaml

```diff
@@ -86,6 +86,10 @@ podLabels: {}
 
 nodeSelector: {}
 
+certificate: {}
+  # defaultKeyAlgorithm
+  # defaultKeySize
```

Comment:
Could you change this to be `defaults.certificates.keyAlgorithm` and add a note about the behaviour?